.github/workflows/test-build.yaml

```diff
@@ -9,8 +9,8 @@ on:
 jobs:
   build_bundle:
     name: build_bundle
-    if: github.repository == 'csi-addons/kubernetes-csi-addons'
```

Comment:
if we remove this won't these builds run on the fork repos?

Reply:
The other `build_*` tests do not have this line. Without any configuration, forks get a single-platform build test.